spec/lib/jobs/validate-ssh-spec.js

```diff
@@ -79,6 +79,15 @@ describe('Validate Ssh', function() {
                 expect(validateSshJob._done).to.be.calledWith(error);
             });
         });
+
+        it('should skip and succeed if no users are defined', function() {
+            validateSshJob.users = null;
+            this.sandbox.stub(validateSshJob, '_done').resolves();
+            return validateSshJob._run()
+            .then(function() {
+                expect(validateSshJob._done).to.be.calledOnce;
```

Comment:
need another assertion to expect to calledWith `undefined`, otherwise, because if changing the code to `_done(err)`, this unit test will still pass.